nodeadm/internal/system/_assets/eks-80-ec2.network.template

```diff
@@ -0,0 +1,24 @@
+[Match]
+Driver=ena ixgbevf vif
+PermanentMACAddress={{.PermanentMACAddress}}
```

Comment:
do we want the snapshot of the entire config to add `PermanentMACAddress`, or does it makes more sense to use a `80-ec2.network.d/*.conf` drop-in?

Reply:
+1, ideally we just patch in the MAC instead of maintaining a copy of the full network unit here

Reply:
@ndbaker1 
the drop-in might be better as it will keep the original ec2's settings. 
my original thought were to use the existence check to have the ability to override this behavior with a custom file(with drop-ins you cannot). But cx can also override this anyway with a higher priority files as well.
let me do the fix